### PR TITLE
Update 1.2

### PR DESCRIPTION
**Pull request by QuoterMusic**
Map Tiles Optimization

Tested up to Wordpress 6.4
- Plugin Requires now Elementor Version 3.5
- Changing the Font Awesome file path (Version number removed)

Update Assets
- Update Leaflet to 1.9.4
- Update Font Awesome to 6.4.2

Elementor Deprecations
- Fix Elementor Deprecations

### CODE REVIEW

**Comment by QuoterMusic** [comment]
@flopperj 

It would be great if you could test the version once before merging.

In my local test environment as well as on a live site everything worked for me.

I tried to find and fix all elementor deprecations which could be responsible for current errors and we should meet all current defaults afterwards.

Additionally I made some adjustments to the map tiles and also brought the attribution to the current state according to the OSM Foundation specifications.

https://osmfoundation.org/wiki/Licence/Attribution_Guidelines

I have also tested the plugin with version 6.4-RC2.

File: README.md
Comment (flopperj) [inline README.md:2]: @QuoterMusic - What are your thoughts of updating the **Open Street Map** link here to **[OpenStreetMap](https://www.openstreetmap.org/copyright)** _(Point to OSM  copyright & license page)_

Comment (flopperj) [reply README.md:2]: Would be great to also to add the same link under the **[License](https://github.com/QuoterMusic/osm-map-elementor/tree/main#licence)** section of the README

Comment (QuoterMusic) [reply README.md:2]: I don't think this is necessary if I would rather suggest the [About page](https://www.openstreetmap.org/about) where you can find all the info about OSM

**Comment by flopperj** [comment]
@QuoterMusic Just tested your latest changes. Everything seems to work as expected! Thanks again for all your contributions. Will wait to approve after you've added the OSM copyright & license link to the README.

File: README.md
Comment (flopperj) [reply README.md:2]: I actually do not have access to the OSM [About page](https://www.openstreetmap.org/about). I can only make edits to the plugin's Wordpress landing page 😃

**Review by flopperj — APPROVED** [review]
LGTM! ✅

**Review by flopperj — APPROVED** [review]

**Comment by QuoterMusic** [comment]
> @QuoterMusic Just tested your latest changes. Everything seems to work as expected! Thanks again for all your contributions. Will wait to approve after you've added the OSM copyright & license link to the README.

**Pull request merged**
